### ISSUE TITLE
why is the resolution=1344x576

### ISSUE THREAD

**Issue by jh-yi** [YanchaoYang/FDA#24]
Hi, 

Is there any reason why the resolution of cityscapes is 1344, 576 instead of 1024, 512 while testing? 

Thanks a lot in advance. I'm looking forward to your reply.

**Comment by wangkaihong** [comment]
Yep, I notice the same thing: it's indicated in the original paper that "... are simply resized to 1024 * 512, with no random cropping...". I actually cannot find any number related to 1344 or 576 in the paper.
